src/pages/index.js

```diff
@@ -80,10 +80,10 @@ class HomeIndex extends React.Component {
                                         <div className="6u 12u$(xsmall)"><input type="email" name="email" id="email" placeholder="Email" /></div>
                                         <div className="12u"><textarea name="message" id="message" placeholder="Message" rows="4"></textarea></div>
                                     </div>
-                                </form>
-                                <ul className="actions">
+                                  <ul className="actions">
```

Comment:
The code is not well indented here.
This `ul` is a sibling to the previous `div`.